addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -4501,19 +4507,14 @@ export class OdooEditor extends EventTarget {
             }
         }
 
-        if (this.options.showEmptyElementHint) {
-            for (const [selector, text] of Object.entries(selectors)) {
-                for (const el of this.editable.querySelectorAll(selector)) {
-                    if (!this.options.isHintBlacklisted(el)) {
-                        this._makeHint(el, text);
-                    }
-                }
-            }
-        }
-
         const block = this.options.getPowerboxElement();
-        if (block) {
-            this._makeHint(block, this.options._t('Type "/" for commands'), true);
+        if (block && this.options.showEmptyElementHint && !this.options.isHintBlacklisted(block)) {
```

Comment:
The fact that you mixed the empty element hints with the powerbox hint has the unintended side effect that now, if `showEmptyElementHint` is false, the `Type "/" for commands` will not be displayed anymore. This wasn't part of the spec. That option should have no effect on `Type "/" for commands` because, from a functional point of view, it is not the same thing as the empty element hints, even if it uses the same mechanism from a technical point of view.

I think maybe it could work by applying my other comment then adding an additional clause? 

That would be:
```js
if (block.nodeName === 'LI' && this.options.showEmptyElementHint) {
    const listMode = getListMode(closestElement(block, 'OL, UL'));
    this._makeHint(block, selectors[listMode], true);
} else if (block.nodeName in selectors && this.options.showEmptyElementHint) {
    this._makeHint(block, selectors[block.nodeName], true);
} else if (block.nodeName === 'P' || block.nodeName === 'DIV') {
    this._makeHint(block, this.options._t('Type "/" for commands'), true);
}
```

Reply:
@dmo-odoo What about `isHintBlacklisted`? Should I add it with `showEmptyElementHint`, or is it not needed anymore?

Reply:
I think the `isHintBlacklisted` condition should apply in all cases.